packages/js/src/redux/selectors/isWooSEO.js

```diff
@@ -9,11 +10,11 @@ import { getIsProduct } from "./editorContext";
 export const getIsWooSeoActive = () => Boolean( get( window, "wpseoScriptData.isWooCommerceSeoActive", false ) );
 
 /**
- * Checks if WooCommerce is active.
+ * Checks if WooCommerce is active. Used also in premium.
```

Comment:
Why specify this? By default all the selectors we add are technically public API.

Reply:
Following the our previous discussion, I thought it could be helpful documentation. But removeing it.